### ISSUE TITLE
Towns by rivers don't have rivers

### ISSUE THREAD

**Issue by Non808** [hasparus/share-azgaar-map#57]
I really like the map generator and this is very much a nitpick. 

If you go to the fantasy city generator from a town near or on top of a river, the town won't have a river in the medieval fantasy city generator. 

For Example:
![screenshot1](https://user-images.githubusercontent.com/34938698/54876808-03b1bf80-4e0e-11e9-8871-ae086ba6b22a.jpeg)
![screenshot2](https://user-images.githubusercontent.com/34938698/54876809-07454680-4e0e-11e9-9f5e-019d276574e4.jpeg)

**Comment by hasparus** [comment]
Hi @Non808! Thanks for the issue and sorry for the late response :)

The problem you describe is in integration between watabou's [City Map Generator](https://github.com/watabou/TownGeneratorOS/) and Azgaar's FMG.

I'd recommend using @Azgaar's deployment instead of my fork. The MFCG works great there. 
I noticed it's improved greatly since I posted https://github.com/Azgaar/Fantasy-Map-Generator/issues/161. It looks like Azgaar fixed your problem few days after you posted your issue here.
